nim_status_client.log

```diff
@@ -0,0 +1 @@
+INF 2021-12-01 22:08:22.930+02:00 Terminating the app as the second instance topics="main" tid=953089 file=nim_status_client.nim:73
```

Comment:
nit: do not commit, maybe add to gitignore

Reply:
meh..removed and added in .gitignore